src/platform/plugins/shared/workflows_management/public/widgets/workflow_yaml_editor/lib/execution_context/build_execution_context.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
Would it be possible to use the correct types instead of disabling the rule for the entire file? Adding exceptions should be the last resort when something can not be typed properly.